generator/internal/rust/templates/common/client_method_preamble.mustache

```diff
@@ -0,0 +1,36 @@
+{{!
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+}}
+    {{#Codec.DocLines}}
+    {{{.}}}
+    {{/Codec.DocLines}}
+    {{#OperationInfo}}
+    ///
+    /// # Long running operations
+    ///
+    /// This method is used to start, and/or poll a [long-running Operation].
+    /// The [Working with long-running operations] chapter in the [user guide]
+    /// covers these operations in detail.
+    ///
+    /// [long-running operation]: https://google.aip.dev/151
+    /// [user guide]: https://googleapis.github.io/google-cloud-rust/
+    /// [working with long-running operations]: https://googleapis.github.io/google-cloud-rust/working_with_long_running_operations.html
+    {{/OperationInfo}}
+    {{#Deprecated}}
+    #[deprecated]
+    {{/Deprecated}}
+    {{#Codec.Attributes}}
+    {{{.}}}
+    {{/Codec.Attributes}}
```

Comment:
I do not think you need (or should) indent these.

Reply:
Thanks, fixed.